unison-src/transcripts/api-namespace-details.output.md

```diff
@@ -39,43 +39,7 @@ Here's a *README*!
 GET /api/non-project-code/namespaces/nested.names
 {
     "fqn": "nested.names",
-    "hash": "#6tnmlu9knsce0u2991u6fvcmf4v44fdf0aiqtmnq7mjj0gi5sephg3lf12iv3odr5rc7vlgq75ciborrd3625c701bdmdomia2gcm3o",
-    "readme": {
-        "contents": [
-            {
-                "contents": "Here's",
-                "tag": "Word"
-            },
-            {
-                "contents": "a",
-                "tag": "Word"
-            },
-            {
-                "contents": {
-                    "contents": [
-                        {
-                            "contents": {
-                                "contents": [
-                                    {
-                                        "contents": "README",
-                                        "tag": "Word"
-                                    }
-                                ],
-                                "tag": "Paragraph"
-                            },
-                            "tag": "Bold"
-                        },
-                        {
-                            "contents": "!",
-                            "tag": "Word"
-                        }
-                    ],
-                    "tag": "Join"
-                },
-                "tag": "Group"
-            }
-        ],
-        "tag": "Paragraph"
-    }
+    "hash": "#sg60bvjo91fsoo7pkh9gejbn0qgc95vra87ap6l5d35ri0lkaudl7bs12d71sf3fh6p23teemuor7mk1i9n567m50ibakcghjec5ajg",
+    "readme": null
```

Comment:
Is this expected?

Reply:
Great catch! Forgot to update the api call to the projects endpoint.

Reply:
Fixed this and all the other api-* ones 👍🏼